cpp/foxglove-websocket/CMakeLists.txt

```diff
@@ -15,9 +15,9 @@ target_link_libraries(foxglove_websocket nlohmann_json::nlohmann_json websocketp
 set_target_properties(foxglove_websocket PROPERTIES CXX_STANDARD 17 CXX_STANDARD_REQUIRED ON)
 
 if (MSVC)
-  target_compile_options(foxglove_websocket PRIVATE /WX /W4)
+  target_compile_options(foxglove_websocket PRIVATE /W4)
 else()
-  target_compile_options(foxglove_websocket PRIVATE -Wall -Wextra -Wpedantic -Werror -Wold-style-cast -Wfloat-equal)
+  target_compile_options(foxglove_websocket PRIVATE -Wextra -Wpedantic -Werror -Wold-style-cast -Wfloat-equal)
```

Comment:
I think the suggestion from the linked discussion was to remove `-Werror`, rather than `-Wall`?

Reply:
:man_facepalming: